src/components/forms/email-template-form.js

```diff
@@ -481,6 +484,14 @@ const EmailTemplateForm = ({ entity, match, errors, clients, preview, templateLo
                                                     </ul>
                                                 </div>
                                                 :
+                                                mjmlRenderError?.message ? 
```

Comment:
beside of showing an error, i do think that we should  not allow saving, like set the save button disabled ?